### DISCUSSION TITLE
How to import multiple .ang files and assign it to different ebsd variables

### DISCUSSION THREAD

**Discussion by vivek9626** [mtex-toolbox/mtex#1374] [Ask Anything] [answered]
Hello all,

I want to import multiple (x50) .ang files and store each .ang file into a different ebsd variable. May I know how I can do that? I saw that I can create an ebsd_list variable somewhere but that did not work for me. Please let me know how I can create an ebsd list for the different .ang files

Regards,
Vivek

**Comment by kilir** [comment, marked as answer]
Hi,
probably doing this in a cell should work (unless you would really need different variables):
```
infiles = dir('*.ang')
for i=1:length(infiles)
ebsd{i} = EBSD.load(infiles(i).name)
end
```
Cheers,
Rüdiger
